packages/amesos2/src/Amesos2_Superludist_TypeMap.hpp

```diff
@@ -69,10 +69,13 @@
 
 #include "Amesos2_TypeMap.hpp"
 
+#ifdef KOKKOS_ENABLE_CUDA
```

Comment:
Is there a way to detect if SLUDist was built with Cuda enabled and add that to this check?

Reply:
That will be nice, but I am not sure. Any idea?

Reply:
Including the cuda headers makes me feel a bit uneasy, would probably be better to guard based on whether sludist was configuring and built with Cuda support but I won't hold up the PR